pkg/exchange/binance/convert.go

```diff
@@ -90,6 +90,132 @@ func toGlobalMarginAccount(account *binance.MarginAccount) *types.MarginAccount
 	}
 }
 
+// Cross
+func toGlobalFuturesAccount(account *futures.Account) *types.FuturesAccount {
+	return &types.FuturesAccount{
+		Assets:                      toGlobalFuturesUserAssets(account.Assets),
+		CanDeposit:                  account.CanDeposit,
+		CanTrade:                    account.CanTrade,
+		CanWithdraw:                 account.CanWithdraw,
+		FeeTier:                     account.FeeTier,
+		MaxWithdrawAmount:           fixedpoint.MustNewFromString(account.MaxWithdrawAmount),
+		Positions:                   toGlobalFuturesUserPositions(account.Positions),//TODO
+		TotalInitialMargin:          fixedpoint.MustNewFromString(account.TotalInitialMargin),
+		TotalMaintMargin:            fixedpoint.MustNewFromString(account.TotalMaintMargin),
+		TotalMarginBalance:          fixedpoint.MustNewFromString(account.TotalMarginBalance),
+		TotalOpenOrderInitialMargin: fixedpoint.MustNewFromString(account.TotalOpenOrderInitialMargin),
+		TotalPositionInitialMargin:  fixedpoint.MustNewFromString(account.TotalPositionInitialMargin),
+		TotalUnrealizedProfit:       fixedpoint.MustNewFromString(account.TotalUnrealizedProfit),
+		TotalWalletBalance:          fixedpoint.MustNewFromString(account.TotalWalletBalance),
+		UpdateTime:                  account.UpdateTime,
+	}
+}
+
+func toGlobalFuturesUserAssets(userAssets []*futures.AccountAsset) (retAssets []types.FuturesUserAsset) {
+	for _, asset := range userAssets {
+		retAssets = append(retAssets, types.FuturesUserAsset{
+			Asset:				asset.Asset,
+			InitialMargin:		fixedpoint.MustNewFromString(asset.InitialMargin),
+			MaintMargin:            fixedpoint.MustNewFromString(asset.MaintMargin),
+			MarginBalance:          fixedpoint.MustNewFromString(asset.MarginBalance),
+			MaxWithdrawAmount:      fixedpoint.MustNewFromString(asset.MaxWithdrawAmount),
+			OpenOrderInitialMargin: fixedpoint.MustNewFromString(asset.OpenOrderInitialMargin),
+			PositionInitialMargin:  fixedpoint.MustNewFromString(asset.PositionInitialMargin),
+			UnrealizedProfit:       fixedpoint.MustNewFromString(asset.UnrealizedProfit),
+			WalletBalance:         fixedpoint.MustNewFromString(asset.WalletBalance),
+		})
+	}
+
+	return retAssets
+}
+
+func toGlobalFuturesUserPositions(userPositions []*futures.AccountPosition) (retAssets []types.FuturesUserPosition) {
+	for _, position := range userPositions {
+		retAssets = append(retAssets, types.FuturesUserPosition{
```

Comment:
we have global position type, called bbgo.Position. let's see if we can extend it later.

Reply:
types/account.go and bbgo/position.go will cause import cycle error. Any idea ?